test/integration_test.go

```diff
@@ -589,7 +590,7 @@ func (s *IntegrationSuite) TestHints() {
 	}
 
 	for _, it := range []tt{
-		{"/*A! master */ SELECT * FROM student WHERE uid = 42 AND 1=2", nil, 0},
+		{"/*A! master */ SELECT * FROM student WHERE score > 100", nil, 0},
```

Comment:
Missing shard key, will fail if table is not allow full-scan.

Reply:
done